package.json

```diff
@@ -73,7 +73,7 @@
     "vrtest": "cd apps && cd vr-tests && yarn screener"
   },
   "devDependencies": {
-    "@azure/data-tables": "12.0.0-beta.3",
+    "@azure/data-tables": "^12.0.0-beta.3",
```

Comment:
dependencies in root package.json should be without carets, since we follow single version policy here